autoresponder.js

```diff
@@ -1,25 +1,21 @@
-const Nexmo = require('nexmo')
-
+const Vonage = require("@vonage/server-sdk");
+const express = require("express");
 const app = require('express')()
```

Comment:
Pretty sure you can do this also since you extracted `express` as a variable. It's what we did in our app-to-phone tutorial
```suggestion
const app = express();
```